dev/packages/alpha/nginx/0.0.3/dataset/ingress_controller/manifest.yml

```diff
@@ -0,0 +1,23 @@
+title: Nginx ingress_controller logs
+release: ga
+type: logs
+streams:
+- input: logs
```

Comment:
Now that ingress and access are both enabled, it looks like we are tailing the same file twice by default?

Reply:
I will change the path to `ingress.log` and disable the ingress_controller dataset.

Reply:
Fixed